### DISCUSSION TITLE
Possible to view data from different Grafana Instance?

### DISCUSSION THREAD

**Discussion by Perkinje** [teslamate-org/teslamate#2443] [Q&A] [answered]
I already have grafana running on another machine. I use it to display data from multiple sources. It's the only grafana I'd like to have running but I'm having trouble exporting/importing the teslamate dashboards into my main grafana (mG) instance. Adding the teslamate datasource worked fine.

After importing each dasboard json, I see multiple errors that it failed to update a legacy series datasource. My mG connects the teslamate postgresql db and can read the data. The problem is the format of the panels is incorrect. Table headers are wrong. Sometimes the column data is formatted as a date or other nonsense. I've rechecked the variables and variable overrides and/or remade them and it doesn't seem to make a difference. The mG version is 8.3.3 running in a TrueNAS jail. The teslamate version is 8.3.4

![Screen Shot 2022-02-05 at 1 52 46 PM](https://user-images.githubusercontent.com/60911845/152660245-ac74725f-cc52-46cb-a6ce-2fec9393e2eb.png)
This image shows the native "Charges" panel.

![Screen Shot 2022-02-05 at 1 53 00 PM](https://user-images.githubusercontent.com/60911845/152660255-abb9ea6e-0bd6-41c7-ae88-58abf1d705c7.png)
This images shows the mG panel. The data in multiple columns are malformed.

I've directly copied and pasted the query. Any help would be greatly appreciated.

**Comment by edasque** [comment, marked as answer]
I've had a few issues, mostly because Teslamate doesn't support Grafana 8.x but nothing like you describe.

I added the data source manually, same name
![image](https://user-images.githubusercontent.com/219187/152680135-711d3516-0e72-4622-8df4-79363e7c03ef.jpeg)

And imported the dashboards by hand. Eventually the manual import proved painful so it set up Grafana provisioning for the dashboards only.

I had to install a few plugins by hand and had to split the Long/Lat query in 2 for the map panels to work but other than that it works splendidly.

I can share my docker compose if that helps.

**Reply by cwanja** [reply]
For what it is worth, TeslaMate was updated to support Grafana v8. https://github.com/adriankumpf/teslamate/releases/tag/v1.26.0
